db/sql/05_msar.sql

```diff
@@ -5765,12 +5765,54 @@ END;
 $$ LANGUAGE plpgsql;
 
 
+CREATE OR REPLACE FUNCTION msar.get_simple_mapping_regclass(join_path jsonb) RETURNS regclass AS $$
+  SELECT (join_path -> 0 -> 1 ->> 0)::bigint;
+$$ LANGUAGE SQL;
+
+
+CREATE OR REPLACE FUNCTION msar.get_simple_mapping_join_cte(
```

Comment:
We can make this into a SQL function instead of plpgsql, as we have `msar.raise_exception` to raise exceptions within SQL functions. But, I think it's fine for now.